packages/design-system-react-native/package.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "@metamask/design-system-react-native",
+  "version": "0.0.0",
+  "description": "Design System React Native",
+  "keywords": [
+    "MetaMask",
+    "Ethereum"
+  ],
+  "homepage": "https://github.com/MetaMask/metamask-design-system/tree/main/packages/design-system-react-native#readme",
+  "bugs": {
+    "url": "https://github.com/MetaMask/metamask-design-system/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/MetaMask/metamask-design-system.git"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "exports": {
+    ".": {
+      "import": "./dist/index.mjs",
+      "require": "./dist/index.js",
+      "types": "./dist/types/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.js",
+  "types": "./dist/types/index.d.ts",
+  "files": [
+    "dist/"
+  ],
+  "scripts": {
+    "build": "ts-bridge --project tsconfig.build.json --verbose --clean --no-references",
+    "build:docs": "typedoc",
```

This will fix the linting issue. I've created an issue to fix the create-package template https://github.com/MetaMask/metamask-design-system/issues/45
```suggestion
    "build:docs": "typedoc",
    "changelog:update": "../../scripts/update-changelog.sh @metamask/design-system-react-native",
```

---

Updated https://github.com/MetaMask/metamask-design-system/pull/44/commits/2157888d595eb7273e50e69edb118b491f6e872d